rskj-core/src/main/java/org/ethereum/core/Repository.java

```diff
@@ -27,7 +27,7 @@
 
 import java.math.BigInteger;
 
-public interface Repository extends RepositorySnapshot {
+public interface Repository extends RepositorySnapshot, TransientRepository {
```

Comment:
Why a repository is both standard repo and transient repo?

Reply:
It's explained a bit above, if still not clear, let me know and I will try to explain more. Maybe we can have a call to discuss.